dotCMS/src/main/java/com/dotcms/graphql/business/RegularContentFieldGenerator.java

```diff
@@ -0,0 +1,27 @@
+package com.dotcms.graphql.business;
+
+import static graphql.schema.GraphQLFieldDefinition.newFieldDefinition;
+import static graphql.schema.GraphQLNonNull.nonNull;
+
+import com.dotcms.contenttype.model.field.Field;
+import graphql.schema.GraphQLFieldDefinition;
+
+/**
+ * This implementation generates a {@link GraphQLFieldDefinition} for any type of {@link Field}
+ * except for {@link com.dotcms.contenttype.model.field.RelationshipField}
+ */
+class RegularContentFieldGenerator implements GraphQLFieldGenerator {
+
+    private ContentAPIGraphQLTypesProvider typesProvider =
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'typesProvider' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642169)